pallets/gas/src/tests.rs

```diff
@@ -419,32 +420,34 @@ fn limit_vs_origin() {
 
         assert_ok!(Gas::create(origin, root_node, 1000));
 
+        assert_ok!(Gas::cut(root_node, cut, 200));
         assert_ok!(Gas::split(root_node, split_1));
         assert_ok!(Gas::split(root_node, split_2));
         assert_ok!(Gas::split_with_value(split_1, split_1_1, 600));
         assert_ok!(Gas::split(split_1, split_1_2));
         assert_ok!(Gas::split(split_1_1, split_1_1_1));
 
-        // Original 1000 less 600 that were `split_with_value`
-        assert_eq!(Gas::get_limit(root_node).unwrap(), Some(400));
+        // Original 1000 less 200 that were `cut` and `split_with_value`
+        assert_eq!(Gas::get_limit(root_node).unwrap(), Some(200));
 
-        // Parent's 400
-        assert_eq!(Gas::get_limit(split_1).unwrap(), Some(400));
+        // Parent's 200
+        assert_eq!(Gas::get_limit(split_1).unwrap(), Some(200));
```

Comment:
could you change initial from 1000 to 1100 to avoid messing up this value (become 300) with the cut one